Sources/hostmgr-helper/VirtualMachineSlot.swift

```diff
@@ -48,11 +46,12 @@ class VirtualMachineSlot: NSObject, ObservableObject, VZVirtualMachineDelegate {
         do {
             let virtualMachine = try await launchConfiguration.setupVirtualMachine()
             virtualMachine.delegate = self
+            self.virtualMachine = ManagedVirtualMachine(machine: virtualMachine, config: launchConfiguration)
 
-            self.virtualMachine = virtualMachine
             try await virtualMachine.start()
```

Comment:
Nitpick: wdyt about renaming the instance variable `self.managedVirtualMachine`, to avoid the potential confusion here between the `let virtualMachine` and `virtualMachine.delegate = self` from line L49–L50 (R47–R48) that refers to a `VZVirtualMachine`, and the `self.virtualMachine` from line R49 that refers to a `ManagedVirtualMachine`… and potentially more significantly, the `try await virtualMachine.start` which confusingly still refers to the `let virtualMachine` and `VZVirtualMachine` instance and not to a `self`-less reference of the `self.virtualMachine =` instance variable above it shadowing it like one could think on first read 😅

Reply:
Sure, yep.